README.md

```diff
@@ -70,24 +70,21 @@ curl https://localhost:3000.com/api/v1/items | jq
       "name": "Carrots",
       "sell_in": 10,
       "quality": 15,
-      "created_at": "2021-05-06T21:31:36.850Z",
-      "updated_at": "2021-05-06T21:31:36.850Z"
+      "description": null
     },
     {
       "id": 2,
```

Comment:
I thought we were not returning the id either?

Reply:
we actually are. We can still discuss whether that is a good idea, but current behavior is to return the id

Reply:
Hmmmmm, ok then